### PR TITLE
Cnft1 2472/patient search list items

### PR DESCRIPTION
**Pull request by rkrusselenq**
## Description

Created a component that takes patient search results to display as a single list view item.
## Tickets

* [Jira Ticket](https://cdc-nbs.atlassian.net/jira/software/c/projects/CNFT1/boards/112?assignee=712020%3A591f5469-427d-49e9-87dc-8b4df72a02b4&assignee=63174e0f3e578bb3b5ffc14a&assignee=63e12176c2b1cb6b347323a8&selectedIssue=CNFT1-2472)

## Checklist before requesting a review
- [x] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

### CODE REVIEW

**Review by alaapbharadwaj — APPROVED** [review]

File: apps/modernization-ui/src/alert/useAlert.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ type AlertProviderProps = {
     children: ReactNode;
 };
 
-function AlertProvider({ duration = 5000, children }: AlertProviderProps) {
+function AlertProvider({ duration = 5000 * 5000, children }: AlertProviderProps) {
```
Comment (stevegsa) [inline apps/modernization-ui/src/alert/useAlert.tsx:27]: was this intentional? 5000 * 5000

File: apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx [hunk 2]
```diff
@@ -1,9 +1,149 @@
...
+                    <br />
+
+                    <span id="driversLicense" className={styles.value}>
+                        {driversLicense?.value ?? noData}
+                    </span>
+                </div>
+
+                <div className={styles.listItemData}>
+                    <label htmlFor="socialSecurityNumber" className={styles.listItemLabel}>
+                        Social Security
```
Comment (hclarkEnq) [inline apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx:136]: question: should we also consider other id types? or are we only keeping these always?

File: apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx [hunk 3]
```diff
@@ -1,9 +1,149 @@
...
+                </div>
+            </div>
+            <div className={styles.listItemBox}>
+                <div className={styles.listItemData}>
+                    <label htmlFor="phone" className={styles.listItemLabel}>
+                        PHONE
+                    </label>
+                    <br />
+                    <span id="phone" className={styles.value}>
+                        {phones[0] ?? noData}
```
Comment (hclarkEnq) [inline apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx:68]: question: are we displaying only one phone number on the list item?

File: apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx [hunk 4]
```diff
@@ -1,9 +1,149 @@
...
+import { Link } from 'react-router-dom';
 
 type Props = {
     result: PatientSearchResult;
 };
 
-const PatientSearchResultListItem = ({ result }: Props) => <>{result.patient}</>;
+const PatientSearchResultListItem = ({ result }: Props) => {
+    const { legalName, birthday, gender, shortId, phones, names, emails, addresses, identification } = result;
+    const noData = 'No Data';
```
Comment (hclarkEnq) [inline apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx:11]: question: should we use the existing <NoData /> component in here?

File: apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx (hunk 2 above)
Comment (adamloup-enquizit) [reply apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx:136]: The designs only show these two identification types, will have to consult with the design team

File: apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx [hunk 5]
```diff
@@ -1,9 +1,149 @@
...
+            <div className={styles.listItemBox}>
+                <div className={styles.listItemData}>
+                    <label htmlFor="legalName" className={styles.listItemLabel}>
+                        LEGAL NAME
+                    </label>
+                    <br />
+                    <Link
+                        id="legalName"
+                        className={`${styles.value}, ${styles.name}`}
+                        to={`patient-profile/${result.patient}/summary`}>
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx:30]: praise:  Much better approach than the current `PatientResult` component that uses a function

File: apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx (hunk 2 above)
Comment (rkrusselenq) [reply apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx:136]: this is what was displayed on the figma so i went with it

File: apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx (hunk 4 above)
Comment (rkrusselenq) [reply apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx:11]: Didn't realize there was an existing component, i'll take a look and see if it matches what we need here, but really this is just a string being stored so i didn't have to type "No data" a bunch of times.

File: apps/modernization-ui/src/alert/useAlert.tsx (hunk 1 above)
Comment (rkrusselenq) [reply apps/modernization-ui/src/alert/useAlert.tsx:27]: that was temporary woops

File: apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx (hunk 3 above)
Comment (rkrusselenq) [reply apps/modernization-ui/src/apps/search/patient/result/list/PatientSearchResultListItem.tsx:68]: I'll follow up with design

**Review by adamloup-enquizit — COMMENTED** [review]
The new search designs do no include multiple examples so it's not clear on what should be displayed.  The existing [PatientResult](https://github.com/CDCgov/NEDSS-Modernization/blob/main/apps/modernization-ui/src/apps/search/patient/result/PatientResult.tsx) is pretty close to the new designs and it also includes some components (`ResultItemLabel`, `ResultItemValue`, and `OrderedData` ) that could be moved out into their own files and used for the other result components.

**Pull request merged**